source/blog/2016-08-29-this-week-in-miq.html.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Last Week in ManageIQ: Speedups, Chargebacks, Toolbars, and more!"
+date: 2016-08-29
+author: jntullo
+comments: true
+published: true
+---
+
+[Jillian](github.com/jntullo) here to guide you through another edition of Last Week in ManageIQ. Let's get this show on the road and explore what we've accomplished this week!
+
+## Featured
+As usual, we had a great week in ManageIQ with 94 pull requests merged from our awesome [contributors](https://github.com/ManageIQ/manageiq/graphs/contributors). Next week, one of them could be you! 
+Head on over to our [issues](https://github.com/manageiq/manageiq/issues) to see where you can contribute - we'd love to have you.
+
+## Improved
+
+### [Index Speedup](https://github.com/ManageIQ/manageiq/pull/10685)
+The _VMs & Instances in My LDAP Group_ index and ajax calls in `/vm_or_template` received an awesome speedup this week
+thanks to the work of [Nick LaMuro](https://github.com/NickLaMuro). Head on over to the PR to view the improvement statistics.
+
+### [Setup and Update Failure Report](https://github.com/ManageIQ/manageiq/pull/10745)
+Thanks to [Joe VLcek](https://github.com/jvlcek), `bin/setup` and `bin/update` will now report any commands that have failed and exit the script. This should be a helpful addition to developers, 
+as they will no longer have to assume that those commands executed correctly.
+
+## Fixed
+
+### [Multi-rate chargeback reports could not be queued](https://github.com/ManageIQ/manageiq/pull/10812)
+Multi-rate chargebacks can now be queued again, as [Libor Pichler](https://github.com/lpichler) resolved the issue by adding in the missing providers.
+
+### [Angular Toolbars in Production](https://github.com/ManageIQ/manageiq/pull/10751)
+Angular toolbars were failing to load in Production mode, but was fixed by [Aparna Karve](https://github.com/AparnaKarve).
```

Comment:
@jntullo 'but was fixed by ' - s/was/were/ - Only because the plural toolbars were fixed - otherwise I couldn't quite put a finger on why I was struggling with the 'was'.